src/v1.js

```diff
@@ -586,6 +586,49 @@ builder.declare({
   res.reply(roles.map(r => roleToJson(r, this)));
 });
 
+/** List role Ids **/
+builder.declare({
+  method:     'get',
+  route:      '/roleids/',
+  query: {
+    continuationToken: /./,
+    limit: /^[0-9]+$/,
+  },
+  name:       'listRoleIds  ',
+  input:      undefined,
+  output:     'list-role-ids-response.yml',
+  stability:  'stable',
+  title:      'List Role IDs',
+  description: [
+    'Get a list role IDs based on the limit and continuationToken',
```

Comment:
`s/list role IDs/list of role IDs/`.

Reply:
My bad, haha. Also, should I remove limit and continuationToken from here as well?